Configuration/Geometry/python/GeometryExtended2023D5Reco_cff.py

```diff
@@ -45,6 +45,10 @@
 from Geometry.CSCGeometryBuilder.idealForDigiCscGeometry_cff import *
 from Geometry.DTGeometryBuilder.idealForDigiDtGeometry_cff import *
 
+#timing
```

Comment:
@lgray - you should modify http://cmslxr.fnal.gov/source/Configuration/Geometry/scripts/dict2023Geometry.py to regenerate this file.